docs/source/TROUBLESHOOTING.rst

```diff
@@ -53,6 +53,18 @@ If that command returns an error, you may not be on Linux. If you are on Linux,
 
 You probably just added yourself to the ``docker`` user group but haven't logged and logged back in yet.
 
+My pods are not autoscaling because the custom metrics are not updating!
+----------------------------
+Prometheus has a large memory footprint, and is liable to be OOMKilled when there are many other pods running.
+
+This can be confirmed by executing the following and inspecting the output for a Reason.
```

Comment:
Why is reason capitalized here?

Reply:
I believe Reason is capitalized in the output but it did take me a while to remember that. I will just leave it as "inspecting the output."